dev/client.scss

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+html,
+body {
+    padding: 0;
+    margin: 0;
+}
+
+#app {
+    display: flex;
+    flex-direction: column;
+}
+
+#textbox,
+#output {
+    padding: 1rem;
+    margin: 0;
+}
+
+#textbox {
+    border: none;
+    background: rgba(0, 0, 0, 0.025);
+}
+
+#output {
+    @import "../styles";
+    @import "../styles/digitalocean";
+
+    overflow: auto;
```

Comment:
Is there a reason for this overflow? Its not doing anything from what I can tell.

Reply:
This is here to ensure that if Markdown content overflows (such as the table being too wide), its the output that will scroll rather than the whole page

Reply:
Yeah that makes sense.